com.github.reduz.ChibiTracker.appdata.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>com.github.reduz.ChibiTracker</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-2.0</project_license>
+  <name>ChibiTracker</name>
+  <summary>ChibiTracker is a portable IT (Impulse Tracker) clone</summary>
+
+  <description>
+    <p>
+      ChibiTracker is a portable IT (Impulse Tracker) clone. It is not a 100% clone though, as it adds more features, such as chorus and reverb, stereo samples, an advanced sample editor and more. The user interface also presents some changes, as it can be custom themed and resized.
+    </p>
+    <p>
+      ChibiTracker is Open Source Software under the GNU Public License, and it can be run under Windows, Mac-OS, Linux, FreeBSD, BeOS, and more to come.
+    </p>
+    <p>
+      For those who haven't used trackers before, ChibiTracker is a small, compact music composing application that is easy to learn and powerful enough to sound good.
+    </p>
+  </description>
+
+​  <launchable type="desktop-id">com.github.reduz.ChibiTracker.desktop</launchable>
+
+  <categories>
+    <category>AudioVideo</category>
+    <category>Audio</category>
+    <category>Sequencer</category>
+  </categories>
+
+  <screenshots>
+    <screenshot type="default">
+      <image type="source">http://content.pouet.net/files/screenshots/00027/00027385.jpg</image>
+    </screenshot>
+  </screenshots>
+
```

Comment:
OARS & release tags are missing.

Reply:
Will do OARS shortly. Not sure if I will be able to do the release tags, since there doesn't seem to be  a "ready" text for that. I can try adapting the message [from commit cf8777b1d3e3b21ea4d3e2e4b33047785cf5c519](https://github.com/reduz/chibitracker/commit/cf8777b1d3e3b21ea4d3e2e4b33047785cf5c519) though.

Reply:
They can be without descriptions.